### DISCUSSION TITLE
Any chance of setting up a discord?

### DISCUSSION THREAD

**Discussion by GR0DZ1LLA** [GerryDazoo/Slinger#133] [General]
I am having hard time with this and I think having some live chat would really help!

This is the topology I'm trying to setup.  The instructions are a little scattered so I'm writing my own and willing to post a guide as to how I did but, just can't get it working.

![image](https://user-images.githubusercontent.com/115742631/195705377-3e6b9187-5557-4c8f-a0a4-591e0918f484.png)

**Comment by GerryDazoo** [comment]
check out my networking guide I posted recently. What you want to do, should be straight forward.

**Reply by GerryDazoo** [reply]
You'll need two config.ini files one for the remote slingbox  aaa.aaa.aaa.aaa:5201 and one for the local slingbox  10.0.30.91:????
On the router in front of the Pi you'll need two port forwarding rule to give access to each slingbox.    8080->10.0.10.90:8080 (the [SERVER] port in one of the config.inis) and one for port 8082.  
Make sure the firewall on the Pi (if there is one running) also has rules to allow tcp connections to port 8080 and 8082

**Reply by GR0DZ1LLA** [reply]
I got the local network running and streaming.  Phew.  How do I create two config.ini on the same server?  config1 and config2?  Do I have to change the server somewhere to make sure it knows there's two configs?  I'm having an issue getting to the remote Slingbox at the top of the diagram.  I have a port forward, I can confirm it's open.  But I can't get the stream.  My RPI server is http://10.0.10.90:8080/slingbox.  When I use the local config, it works to get to the 30.91 Slingbox.  When I replace the config and restart the service with the variables to see the 1.254 Slingbox, it just tries and tries.  I'm also realizing I'm going to have to get some kind of domain or dynamic DNS because both the server location and the Slingbox remote location both have consumer ISPs (the IPs are subject to change).

**Reply by GR0DZ1LLA** [reply]
Oddly enough, on the remote Slingbox, I can kick myself off.  I get the notice that someone else has connected but it never shows me the stream.

**Reply by GR0DZ1LLA** [reply]
I got it working.  Now I have to figure out Dynamic DNS issues and the remotes!

**Reply by GerryDazoo** [reply]
Yes two separate files whatever name you want and you pass them in on the command line. If you're running the sling.service on the Pi you need to edit the service definition file and fix the ExecStart line to include config file names.
Make sure the [SERVER] ports in each config are different by at least two (8080 and 8082 for example) and you'll need port forwarding rules on the Pi router for each of them.

**Comment by colinpeddle** [comment]
Echoing the desire for a discord so I made one because I'm assuming it's low on the list of priorities for Gerry and this project

https://discord.gg/rMsMepD3c5

@GerryDazoo whenever you get around to it, the server is of course all yours.
